gaintrain-fe/components/cycles/cycles-card.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import BaseCard from '../base-components/base-card';
+import BaseTag from '../base-components/base-tag';
+
+const TableContainer = styled.View`
+   display: flex;
+   flex-direction: row;
+   width: 100%;
+`;
+
+const TableColumn = styled.View`
+   display: flex;
+   flex-direction: column;
+   width: ${(props) => props.widthPercentage};
+`;
+
+const Row = styled.View`
+   display: flex;
+   flex-direction: row;
+`;
```

Comment:
This isn't something you need to fix in this PR, but just pointing out for me to remember to do. We should probably make this a CSS class where we can just apply it to views. That way we don't have to keep re-making this `Row` styled-component